check_test.go

```diff
@@ -404,7 +407,7 @@ func getCheckTestCases() map[string]TmpCheckTestCase {
 			fixture: BuildUpdateRequest("HasRecentDeploy", map[string]any{"days": 5}),
 			body: func(c *ol.Client) (*ol.Check, error) {
 				input := ol.NewCheckUpdateInputTypeOf[ol.CheckHasRecentDeployUpdateInput](checkUpdateInput)
-				input.Days = ol.RefOf(5)
+				input.Days = ol.NewNullableFrom(5)
```

Comment:
heres an `int` where i'm not sure `null` makes sense - can we test the API in Graphiql?